introduction.md

```diff
@@ -0,0 +1 @@
+Hello i'm Wim working for buildnet, i'm just follow the course github on codeschool and have to do this. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.